src/main/java/com/opencart/steps/SearchProductPageBL.java

```diff
@@ -0,0 +1,41 @@
+package com.opencart.steps;
+
+import com.opencart.pages.SearchProductPage;
+import com.opencart.pages.SuccessSearchProductResultPage;
+import org.testng.Assert;
+
+/*
+* Class that describes steps (Business logic) of searching product
+* with correct search criteria "mac"
+ */
+
+public class SearchProductPageBL {
+
+    private SearchProductPage searchProductPage;
+    private SuccessSearchProductResultPage successSearchProductResultPage;
+
+    public SearchProductPageBL() {
+        searchProductPage = new SearchProductPage();
+    }
+
+    public SearchProductPageBL makeSearch() {
```

Comment:
Do that method more flexible for different search phrases. Just put phrase as a parameter of this method.